core-bundle/contao/templates/twig/component/_rich_text.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+    This component outputs rich text content, that was typically composed by
+    an editor in the backend using the tinyMCE editor.
+
+      <div class="rte">
+        <p>Lorem ipsum</p>
+      </div>
+
+    Mandatory variables:
+        @var string text
```

Comment:
Should we add a note that `text` gets output `|raw` and therefore needs to be trusted?

Reply:
Good point, added in 4b3dc38.